Components/RegisterScreen.js

```diff
@@ -1,77 +1,112 @@
-import React from 'react';
+import React, {Component} from 'react';
 import { StyleSheet, TextInput, Text, View, Button, Image } from 'react-native';
 import { createStackNavigator } from 'react-navigation';
 import logo from './images/logo.png';
+import { connect } from 'react-redux';
+import {submitNewUserInformation} from './helperFunctions/Login';
+import {updateUserObject, updateIsUserLoggedIn} from '../actions';
 
-// let submitUserLoginInformation = (event, dispatch, history) => {
-//     event.preventDefault();
-//     let email = event.target.email.value;
-//     let password = event.target.password.value;
-//     console.log(email, password)
-//     event.target.reset();
-// };
+class RegisterScreenDumb extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { email: '', password: '' };
+      }
 
-let RegisterScreen = ({props}) =>
-<View style={styles.container}>
+    render() {
+        let {navigation, dispatch} = this.props;
+        let {email, password} = this.state;
+
+        let registerUser = () => {
```

Comment:
i dont really to understand this part.

Reply:
Ok, I thought that we were using email for the username.

Which part do you not understand?  Local state (the component state) is only saving the value of the input fields and then resets it once the submit button is pressed.  The redux store (this.props) is saving the navigation props and the dispatch actions.